mmda/types/box.py

```diff
@@ -12,8 +12,8 @@
 
 def is_overlap_1d(start1: float, end1: float, start2: float, end2: float) -> bool:
     """Return whether two 1D intervals overlaps"""
-    assert start1 < end1
-    assert start2 < end2
+    assert start1 <= end1
```

Comment:
Zero-width bounding boxes cause an error at this line. Is it an error to produce these? I'm inclined to say they're okay in principle. If we don't want to allow them, they should be detected/removed at some earlier point.